### ISSUE TITLE
Element attributes not updated in profile component after been changed in the base

### ISSUE THREAD

**Issue by carolinerosin** [haffonist/igamt#1644]
If I change an attribute (for example cardinality) in my base element, it should also change in the profile component.

Example:
I create a MSH flavor (let's call it MSH_1). MSH-3 cardinality is [0..1], datatype HD 
I create a GU profile component, add the element "MSH-3" from segment level and change the datatype to the "HD_GU" flavor.

Then I go back to my MSH_1 flavor and change the cardinality of MSH-3 to [0..2].
When I go back to my GU profile component definition, the item "MSH_1.3" is still [0..1], it hasn't been updated to [0..2] as expected.

**Comment by afrold** [comment]
@robnist what do you say ? I remembered we already said no to this.

**Comment by carolinerosin** [comment]
@haffonist I talked to Rob and here's what he thinks.
When the user wants to "view" the profile component, only what has changed should be displayed.
When the user wants to "edit" the profile component, the user should see what comes from the base element and what was changed (in yellow). If something changes in the base element, it should be reflected in the "edit" view. 

Let me know if you need more details.

**Comment by carolinerosin** [comment]
@haffonist I just came across this old ticket, this is what we discussed yesterday

**Comment by afrold** [comment]
Yes. Refreshing the attributes will not be needed in a future version of the profile components. Subsequent discussion will follow prior to implementation
